tests/test_zarr_metadata.py

```
@@ -66,8 +70,10 @@ def test__given_a_valid_bucket_name_and_bucket_does_not_contain_collections__the
             self, mocker, s3_fs
         ):
             mocker.patch(
-                "dclimate_zarr_client.zarr_metadata.get_catalog_metadata",
-                return_value=json.loads(open(METADATA / "Empty Data Catalog.json").read()),
+                "src.zarr_metadata.get_catalog_metadata",
+                return_value=json.loads(
+                    open(METADATA / "Empty Data Catalog.json").read()
```

Ruff (0.8.2)

Use a context manager for opening files

(SIM115)